docs/issues_catched.md

```diff
@@ -0,0 +1,34 @@
+
+# Issues Precaution finds
+
+Precaution utilizes multiple security static code analysis tools.
+That is the reason why the amount and types of checks for the different
+programming languages differs.
+The security static code analyzers are language-focused
+and some of the vulnerabilities they find are specific 
+for the programming language.
+However, some of those rules can be grouped:
+
+* Hardcoded credentials
+* SQL injections
+* Command injections
+* HTML injections
+* Weak random number generator
+* Imported and used weak cryptographic algorithms
+* Imported insecure libraries
+* Use of specific insecure functions from libraries
+* Weak key used for a cryptographic algorithm
+* Eval function used
```

Comment:
Is that all uses of `eval` or just uses which call `eval` on user-supplied data?

Reply:
In Bandit is whenever the function `eval` is used https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b307-eval
in TSLint-config-security is whenever `eval` is used with a non-literal argument: 
https://github.com/webschik/tslint-config-security#tsr-detect-eval-with-expression